generate-docs/scripts/preprocessor.ts

```diff
@@ -102,12 +102,18 @@ tryCatch(async () => {
         handleCommonImports(handleLiteralParameterOverloads(dtsBuilder.extractDtsSection(releaseDefinitions, "Begin OneNote APIs", "End OneNote APIs")), "Other")
     );
 
-    console.log("create file: outlook.d.ts");
+    console.log("create file: outlook.d.ts (preview)");
     fsx.writeFileSync(
         '../api-extractor-inputs-outlook/outlook.d.ts',
         handleCommonImports(dtsBuilder.extractDtsSection(previewDefinitions, "Begin Exchange APIs", "End Exchange APIs"), "Outlook")
     );
 
+    console.log("create file: outlook.d.ts (release)");
+    fsx.writeFileSync(
+        '../api-extractor-inputs-outlook-release/Outlook_1_7/outlook.d.ts',
```

Comment:
not sure if casing matters but should it instead be:
```suggestion
        '../api-extractor-inputs-outlook-release/outlook_1_7/outlook.d.ts',
```